data/result-exporter/cne-exporter-commons/src/main/java/com/powsybl/openrao/data/cneexportercommons/CneUtil.java

```diff
@@ -76,4 +92,84 @@ public static String generateUUID() {
+        if (mnecAcceptableMarginDiminution != 0) {
+            MnecParametersExtension mnecParametersExtension = new MnecParametersExtension();
+            mnecParametersExtension.setAcceptableMarginDecrease(mnecAcceptableMarginDiminution);
+            raoParameters.addExtension(MnecParametersExtension.class, mnecParametersExtension);
+        }
+        return raoParameters;
+    }
+
+    private static ObjectiveFunctionParameters.ObjectiveFunctionType getObjectiveFunctionTypeFromString(String objectiveFunctionType) {
+        if ("max-min-relative-margin-in-ampere".equals(objectiveFunctionType)) {
+            return ObjectiveFunctionParameters.ObjectiveFunctionType.MAX_MIN_RELATIVE_MARGIN_IN_AMPERE;
+        } else if ("max-min-relative-margin-in-megawatt".equals(objectiveFunctionType)) {
+            return ObjectiveFunctionParameters.ObjectiveFunctionType.MAX_MIN_RELATIVE_MARGIN_IN_MEGAWATT;
+        } else if ("max-min-margin-in-ampere".equals(objectiveFunctionType)) {
+            return ObjectiveFunctionParameters.ObjectiveFunctionType.MAX_MIN_MARGIN_IN_AMPERE;
+        } else if ("max-min-margin-in-megawatt".equals(objectiveFunctionType)) {
+            return ObjectiveFunctionParameters.ObjectiveFunctionType.MAX_MIN_MARGIN_IN_MEGAWATT;
+        } else {
+            throw new OpenRaoException("Unknown ObjectiveFunctionType %s".formatted(objectiveFunctionType));
+        }
+    }
+
+    public static CneExporterParameters getParametersFromProperties(Properties properties) {
+        return new CneExporterParameters(
+            getPropertyOrThrowException(properties, DOCUMENT_ID),
+            Integer.parseInt(getPropertyOrThrowException(properties, REVISION_NUMBER)),
+            getPropertyOrThrowException(properties, DOMAIN_ID),
+            getProcessTypeFromString(getPropertyOrThrowException(properties, PROCESS_TYPE)),
+            getPropertyOrThrowException(properties, SENDER_ID),
+            getRoleTypeFromString(getPropertyOrThrowException(properties, SENDER_ROLE)),
+            getPropertyOrThrowException(properties, RECEIVER_ID),
+            getRoleTypeFromString(getPropertyOrThrowException(properties, RECEIVER_ROLE)),
+            getPropertyOrThrowException(properties, TIME_INTERVAL)
+        );
+    }
+
+    private static String getPropertyOrThrowException(Properties properties, String propertyName) {
+        if (properties.containsKey(propertyName)) {
+            return properties.getProperty(propertyName);
+        }
+        throw new OpenRaoException("Could not parse CNE exporter parameters because mandatory property %s is missing.".formatted(propertyName));
+    }
+
+    private static CneExporterParameters.ProcessType getProcessTypeFromString(String processType) {
+        switch (processType) {
+            case "A48" -> {
+                return CneExporterParameters.ProcessType.DAY_AHEAD_CC;
```

Comment:
enums ProcessType & RoleType should be deleted IMO, since they are defined by entsoe. gridcapa should put that in their code. what do you think @phiedw ?

Reply:
I deleted the whole CneExporterParameters class with its enums